src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -1,11 +1,34 @@
 package core.mate.academy.model;
 
+import java.util.Random;
+
 /**
  * Add some custom fields that could be only in Bulldozer
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private int number;
+    private int year;
+
     public Bulldozer() {
+        this.number = new Random().nextInt(100);
+        this.year = (int)Math.floor(Math.random() * (2022 - 1960 + 1) + 1960);
```

Comment:
you should pass already calculated values to the constructor, the same for other machines